pkg/await/await.go

```diff
@@ -42,22 +42,36 @@ import (
 // --------------------------------------------------------------------------
 
 const (
-	appsV1Deployment                     = "apps/v1/Deployment"
-	appsV1Beta1Deployment                = "apps/v1beta1/Deployment"
-	appsV1Beta2Deployment                = "apps/v1beta2/Deployment"
-	autoscalingV1HorizontalPodAutoscaler = "autoscaling/v1/HorizontalPodAutoscaler"
-	extensionsV1Beta1Deployment          = "extensions/v1beta1/Deployment"
-	storageV1StorageClass                = "storage.k8s.io/v1/StorageClass"
-	coreV1ConfigMap                      = "v1/ConfigMap"
-	coreV1LimitRange                     = "v1/LimitRange"
-	coreV1Namespace                      = "v1/Namespace"
-	coreV1PersistentVolume               = "v1/PersistentVolume"
-	coreV1PersistentVolumeClaim          = "v1/PersistentVolumeClaim"
-	coreV1Pod                            = "v1/Pod"
-	coreV1ReplicationController          = "v1/ReplicationController"
-	coreV1ResourceQuota                  = "v1/ResourceQuota"
-	coreV1Secret                         = "v1/Secret"
-	coreV1Service                        = "v1/Service"
+	appsV1Deployment                            = "apps/v1/Deployment"
+	appsV1Beta1Deployment                       = "apps/v1beta1/Deployment"
+	appsV1Beta2Deployment                       = "apps/v1beta2/Deployment"
+	autoscalingV1HorizontalPodAutoscaler        = "autoscaling/v1/HorizontalPodAutoscaler"
+	extensionsV1Beta1Deployment                 = "extensions/v1beta1/Deployment"
+	storageV1StorageClass                       = "storage.k8s.io/v1/StorageClass"
+	coreV1ConfigMap                             = "v1/ConfigMap"
+	coreV1LimitRange                            = "v1/LimitRange"
+	coreV1Namespace                             = "v1/Namespace"
+	coreV1PersistentVolume                      = "v1/PersistentVolume"
+	coreV1PersistentVolumeClaim                 = "v1/PersistentVolumeClaim"
+	coreV1Pod                                   = "v1/Pod"
+	coreV1ReplicationController                 = "v1/ReplicationController"
+	coreV1ResourceQuota                         = "v1/ResourceQuota"
+	coreV1Secret                                = "v1/Secret"
+	coreV1Service                               = "v1/Service"
+	coreV1ServiceAccount                        = "v1/ServiceAccount"
+	extensionsV1Beta1Ingress                    = "extensions/v1beta1/Ingress"
+	rbacAuthorizationV1ClusterRole              = "rbac.authorization.k8s.io/v1/ClusterRole"
```

Comment:
I'm sure you were intentional about this, and that this is just flowing through the underlying Kubernetes object model, but for my education, why the fully qualified name, versus just `rbac/v1/...`?

Reply:
The API server does not understand `rbac/v1/...`. It requires a fully-qualified group name in this case.